egs2/librispeech/asr1/local/make_lexicon_fst.py

```diff
@@ -0,0 +1,411 @@
+#!/usr/bin/env python3
```

Comment:
Ditto.
We can move to https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE/asr1/scripts In the future.